packages/test-utils/src/ConditionalTest.ts

```diff
@@ -29,14 +29,6 @@ export function skipSuiteOnJestCircus(): void {
   }
 }
 
-export function skipSuiteOnWindows(): void {
```

Comment:
Keeping this until all skips are resolved seems better to me - easier to find usage 🙂

Reply:
It was in part for the excitement of it - the two tests that are remaining don't seem like they're going to go away anytime soon (the yarn pnp one at least), and I felt that removing this helper made it harder to justify skipping future tests in Windows.

Happy to add it back 🙂

Reply:
aight, consider me convinced! 🙂